generative_data_prep/__main__.py

```diff
@@ -297,6 +294,11 @@ def get_categories(categories_path: str):
         raise ValueError(err_msg)
     verify_input_file(args.input_file_path)
     output_dir = get_output_dir(args.cmd, args.output_path, args.overwrite_output_path)
+    Logger.add_file_handler(args.log_file_path, output_dir)
```

Comment:
in `main` we can simply have this. Defining a `Logger` is unnecessary and reduces code maintainability.

```
import logging.config
logging.config.fileConfig('/path/to/logging.conf')

Reply:
Main reason to have a wrapper class is if we want to add additional wrapper functionality to our logging functions: https://github.sambanovasystems.com/SambaNova/software/blob/main/sambaflow/samba_tools/samba_tools/logging/logging.py#L192.  This is similar to what was done in SambaTools.  Agree that this is not necessary today, but in the future we might have to build a wrapper class to add these additional functionality.

Reply:
For now the main wrapper functionality is adding the SEP_STR constant to specific messages, which isn't a big deal for now.

Reply:
So I agree in the short term we can use just the base logging functionality provided in Python, but there's a good chance we will have to port this later on to a class.

Reply:
I see what you mean. For now, I think we should refrain from trying to retrofit new implementations to our legacy implementations. We will cross that bridge when we get there. If anything, we are moving towards modernizing legacy systems and they will follow these new low-maintenance implementations as we migrate them to modular and leaner applications.

Reply:
Done!